batched/dense/impl/KokkosBatched_Pttrs_Serial_Impl.hpp

```diff
@@ -0,0 +1,101 @@
+//@HEADER
+// ************************************************************************
+//
+//                        Kokkos v. 4.0
+//       Copyright (2022) National Technology & Engineering
+//               Solutions of Sandia, LLC (NTESS).
+//
+// Under the terms of Contract DE-NA0003525 with NTESS,
+// the U.S. Government retains certain rights in this software.
+//
+// Part of Kokkos, under the Apache License v2.0 with LLVM Exceptions.
+// See https://kokkos.org/LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//@HEADER
+#ifndef KOKKOSBATCHED_PTTRS_SERIAL_IMPL_HPP_
+#define KOKKOSBATCHED_PTTRS_SERIAL_IMPL_HPP_
+
+#include <KokkosBatched_Util.hpp>
+#include <KokkosBlas1_scal.hpp>
+#include "KokkosBatched_Pttrs_Serial_Internal.hpp"
+
+/// \author Yuuichi Asahi (yuuichi.asahi@cea.fr)
+
+namespace KokkosBatched {
+
+template <typename DViewType, typename EViewType, typename BViewType>
+KOKKOS_INLINE_FUNCTION static int checkPttrsInput(
+    [[maybe_unused]] const DViewType &d, [[maybe_unused]] const EViewType &e,
+    [[maybe_unused]] const BViewType &b) {
+  static_assert(Kokkos::is_view<DViewType>::value,
```

Comment:
This is fine but if you wanted to use the shorter: `Kokkos::is_view_v<DViewType>` that's also okay and saves you a bit of time